tardis/adapters/sites/htcondor.py

```diff
@@ -70,9 +74,18 @@ def __init__(self, machine_type, site_name):
         submit_jdl = self.configuration.MachineTypeConfiguration[self._machine_type].jdl
         drone_resources = ";".join(
             [f'TardisDrone{resource}={self.machine_meta_data[resource]}' for resource in self.machine_meta_data])
+        submit_resources_args = ''
+        for resource in self.machine_meta_data:
```

Comment:
Two iterations over self.machine_meta_data. Line 76 and Line 78. Please, refactor!